tests/v2/validation/Jenkinsfile.registries

```diff
@@ -0,0 +1,140 @@
+#!groovy
+node {
+    def rootPath = "/root/src/github.com/rancher/rancher/"
+    def workPath = "/root/src/github.com/rancher/rancher/tests/v2/validation/"
+    def job_name = "${JOB_NAME}"
+    if (job_name.contains('/')) { 
+      job_names = job_name.split('/')
+      job_name = job_names[job_names.size() - 1] 
+    }
+    def golangTestContainer = "${job_name}${env.BUILD_NUMBER}_golangtest"
+    def imageName = "rancher-registries-validation-${job_name}${env.BUILD_NUMBER}"
+    def testsDir = "/root/src/github.com/rancher/rancher/tests/v2/validation/${env.TEST_PACKAGE}"
+    def testResultsOut = "results.xml"
+    def envFile = ".env"
+    def rancherConfig = "rancher_env.config"
+    def branch = "release/v2.6"
```

Comment:
> Another nit, I think the default should be 2.7 since this PR points to 2.7
This is pointing to 2.6 a forward port will be created once reviews are done.